tests/hardhat/XVS/XVSVaultFix.ts

```diff
@@ -0,0 +1,218 @@
+import { impersonateAccount, reset } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { Signer } from "ethers";
+import { ethers } from "hardhat";
+
+import { XVSVaultProxy__factory, XVSVault__factory } from "../../../typechain";
+
+const hre = require("hardhat");
+const FORK_MAINNET = process.env.FORK_MAINNET === "true";
+let FORK_ENDPOINT;
+
+const poolId = 0;
+const vaultProxy = "0x051100480289e704d20e9DB4804837068f3f9204";
+const affectedUserAddress = "0xddbc1841be23b2ab55501deb4d6bc39e3f8aa2d7";
+const affectedUserAddress2 = "0x3c7ea3ae7c47bd817c63c47e9ecee89452471a9e";
+const affectedUserAddress3 = "0x37d768c8fc5a0f54754a6bdb8b8469e6ff8cad07";
+const vaultUser = "0xc09a9a0533a0b247c8bb672b2d37cd2c58394768";
+const Owner = "0x1c2cac6ec528c20800b2fe734820d87b581eaa6b";
+const tokenAddress = "0xcf6bb5389c92bdda8a3747ddb454cb7a64626c63";
+let admin: Signer;
+let vaultUserSigner: Signer;
+let signer: Signer;
+let xvsVault;
+let oldXVSVault;
+
+const beforeUpgrade = [
```

Comment:
I would move this into the test "Verify Expected Result old withdraw requests for affectedUser1" because it's the only place where we are using it, just to improve the readability of the code.